src/documentmanager/documentmanager.cpp

```diff
@@ -381,7 +385,7 @@ QMdiArea* DocumentManager::getArea(CodeEditor *doc)
         areaIter = std::find_if(subWdwList.cbegin(), subWdwList.cend(),
                                 [&doc](const auto& document)
         {
-            return static_cast<CodeEditor*>(document->widget()) == doc;
+            return qobject_cast<CodeEditor*>(document->widget()) == doc;
```

Comment:
`qobject_cast<CodeEditor*>(document->widget()` can return nullptr and after that you're comparing this with `doc`. Check it

Reply:
fixed